client/my-sites/domains/domain-management/components/outbound-transfer-confirmation/index.jsx

```diff
@@ -0,0 +1,178 @@
+	};
+
+	constructor( props ) {
+		super( props );
+		this._isMounted = false;
+	}
+
+	componentDidMount() {
+		this._isMounted = true;
+	}
+
+	componentWillUnmount() {
+		this._isMounted = false;
+	}
+
+	setStateIfMounted( ...args ) {
+		if ( this._isMounted ) {
+			this.setState( ...args );
+		}
+	}
+
+	state = {
+		isCanceling: false,
+		isAccepting: false,
+	};
+
+	isRequesting = () => {
+		return this.state.isAccepting || this.state.isCanceling;
+	};
+
+	onAcceptTransferClick = () => {
+		const { domain, translate } = this.props;
+		this.setState( { isAccepting: true } );
+		acceptTransfer( domain.name, error => {
+			this.setStateIfMounted( { isAccepting: false } );
+			if ( error ) {
+				notices.error( this.getErrorMessage( error ) );
+			} else {
+				notices.success(
+					translate(
+						'You have successfully expedited your domain transfer. There is nothing else you need to do.'
+					)
+				);
+			}
+		} );
+		this.props.recordTracksEvent( 'calypso_outbound_transfer_accept_click' );
+	};
+
+	onCancelTransferClick = () => {
+		const { domain, siteId, translate } = this.props;
+		this.setState( { isCanceling: true } );
+		cancelTransferRequest(
+			{
+				domainName: domain.name,
+				siteId: siteId,
+				declineTransfer: true,
+			},
+			error => {
+				this.setStateIfMounted( { isCanceling: false } );
+				if ( error ) {
+					notices.error( this.getErrorMessage( error ) );
+				} else {
+					notices.success(
+						translate(
+							'You have successfully cancelled your domain transfer. There is nothing else you need to do.'
+						)
+					);
+				}
+			}
+		);
+		this.props.recordTracksEvent( 'calypso_outbound_transfer_cancel_click' );
+	};
+
+	getErrorMessage( error ) {
+		const { translate } = this.props;
+
+		switch ( error.error ) {
+			case 'no_pending_transfer':
+				return translate(
+					'This domain is no longer awaiting transfer. If your transfer has completed successfully or ' +
+						'you previously cancelled your domain transfer, there is nothing else you need to do.'
+				);
+			case 'domain_registration_unavailable':
+			case 'tld-in-maintenance':
+				return getMaintenanceMessageFromError( error, translate );
+			default:
+				return error.message;
+		}
+	}
+
+	render() {
+		const { domain, translate } = this.props;
+
+		if ( ! domain.pendingTransfer ) {
+			return null;
+		}
+
+		if ( domain.currentUserCanManage ) {
+			return (
+				<>
+					<p>
+						{ translate(
+							'We received a notification that you would like to transfer this domain to another registrar. ' +
+								'Accept the transfer to complete the process or cancel it to keep your domain with ' +
+								'WordPress.com.'
+						) }
+					</p>
+
+					<p>
+						<Button
+							primary
+							busy={ this.state.isAccepting }
+							disabled={ this.isRequesting() }
+							className="outbound-transfer-confirmation__accept-transfer"
+							onClick={ this.onAcceptTransferClick }
+						>
+							{ translate( 'Accept transfer' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'Accept Transfer' )` *ES Score: 11*